helm/g2p-sandbox/values.yaml

```diff
@@ -146,7 +155,20 @@ ph-ee-engine:
     ams_local_customer_host: "https://fynams.sandbox.mifos.io/"
     ams_local_auth_host: "https://fynams.sandbox.mifos.io/"
     ingress:
-      enabled: false    
+      enabled: true
```

Comment:
Parent charts shouldn't have ingresses enabled as true.

Reply:
Disabled Ingresses for all the services